app/styles/_forms.less

```diff
@@ -1,3 +1,86 @@
+// Inline form row with two inputs and remove and move button controls
+// ----------------------------------------------------
+
+// <ng-form ng-class="{ 'has-sort' : ... ">
+//   <div class="row form-row-has-controls input-labels">
+//     <div class="col-xs-6">
+//       <label class="input-label"> ...
+//     <div class="col-xs-6">
+//       <label class="input-label"> ...
+//   <div class="row form-row-has-controls has-label">
+//     <div class="form-group col-xs-6"> ...
+//     <div class="form-group col-xs-6"> ...
+//     <div class="form-row-controls">
+//       <button class="sort-row"> ...
+//       <button class="btn-remove close delete-row"> ...
+
+.as-sortable-dragging {
+  margin-left: @form-row-controls-button-offset-position;
+}
+.as-sortable-placeholder {
+  border-top: @as-sortable-placeholder-border-top-width @as-sortable-placeholder-border-top-style @as-sortable-placeholder-border-top-color;
+  clear: left;
+  width: @as-sortable-placeholder-width;
+}
+
+.form-row-has-controls {
+  position: relative;
+  width: calc(~"100% - 15px");
+  @media(min-width: @screen-sm-min) {
+    // single input, align right input edge with dual input at wider viewports
+    width: ~"calc(100% - @{form-row-has-controls-width-offset})";
+  }
+  .has-sort &,
+  .as-sortable-dragging & {
+    width: ~"calc(100% - @{form-row-has-controls-width-offset})"; // dual inputs
+    .form-row-controls {
+      // offset by width of two buttons and 8px gutter
+      right: -@form-row-controls-button-offset-position; // 60px
+    }
+  }
+  // Two column padding adjustments to keep associated inputs close together
+  > [class*="col-xs-"]:nth-child(1n) {
+    padding-right: 0;
+  }
+  > [class*="col-xs-"]:nth-child(2n) {
+    padding-left: @form-row-control-button-gutter;
+    padding-right: 0;
+  }
+  .form-row-controls {
+    position: absolute;
+    // offset by width of one button and 8px gutter
+    right: -(@form-row-control-button-width + @form-row-control-button-gutter); // 34px
+    top: 0;
+    button.close,
+    button.sort-row {
+      background: transparent;
+      border: 0;
+      height: @form-row-control-button-height;
+      line-height: 1;
+      // match pf .close rule https://github.com/patternfly/patternfly/blob/master/src/less/close.less#L11
+      opacity: @as-sortable-dragging-opacity;
+      &:focus,
+      &:hover {
+        opacity: @as-sortable-item-icon-hover-opacity;
+      }
+      width: @form-row-control-button-width;
```

Comment:
Nit:  move these to line 62 so the new rule currently there doesn't interrupt this block of declarations.